### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by susangoga**
# Dear reviewer:

## 📌Project: Desktop version.

*_These are the requirements I follow to complete this project for my portfolio website:_*

1.- I use media queries in my CSS to display on desktop screens.
2.- I use Grid and Flexbox.
3.- I replicate the design as much as possible with CSS.

Thanks for your feedback! 👍

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Zuz4n , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ]  Please make sure your desktop version is fully responsive but as you can see from the video below, you can see there is white space at the right when adjusting the responsiveness of the desktop version. If you can not see it in the video try adjusting your desktop version responsiveness to see it.
  
  [Screencast from 11-04-2022 12:59:43 AM.webm](https://user-images.githubusercontent.com/92303891/199863576-f9b6d9d0-c0bf-4de9-b107-f77272074e77.webm)

- [ ] Please from the project requirements, you should stick to the design as much as possible. But as you can see you have some differences in the design. Your design Form text (`I'm always ..` should not align at the center and your form design does not match the figma design You can see the design below and compare it with your project design. :pray:

| Your Design) |Figma Design |
| :---:   | :---: | 
| Screenshot from 2022-11-04 01-12-11  | Screenshot from 2022-11-04 01-02-31 |

- [ ] Please kindly make sure your project is fully responsive. But your project has a horizontal scroll on desktop screens. You can see the screenshot below. :pray:
  Screenshot from 2022-11-04 01-26-25


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html [hunk 1]
```diff
@@ -13,206 +12,222 @@
 <body>
```
Comment (Nyame-Wolf) [inline Index.html:12]: As highlighted by the previous reviewer, I suggest that you  fix your project to fit the entire width of the screen. This is your project currently
Screenshot 2022-11-04 at 08 33 29
Screenshot 2022-11-04 at 08 33 37
I suggest you add add `margin:0` property to your body to override system default.
Screenshot 2022-11-04 at 08 26 32
As you can see in the above screenshot, the system has its own default which you will need to override by adding the properties to body

File: Index.html [hunk 2]
```diff
@@ -13,206 +12,222 @@
...
-            <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-          <div class="list_programs2">
-            <ul>
-              <li<a><img class="htm"src="./images/html2.svg"></a></li> 
-                <li<a><img class="bot"src="./images/boot2.svg"></a></li>
-                  <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+      </div>
+
+
+        <div class="gridprojects">
```
Comment (Nyame-Wolf) [inline Index.html]: - [ ] [OPTIONAL]Kindly change this to a `section` tag. It is the correct semantic and screen readers know where it starts. a `div` is too general

File: Index.html [hunk 3]
```diff
@@ -13,206 +12,222 @@
...
-      <div class="projects">
-        <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
-          <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        
-          <div class="list_programs2">
-          <ul>
-            <li<a><img class="htm"src="./images/html2.svg"></a></li> 
-              <li<a><img class="bot"src="./images/boot2.svg"></a></li>
-                <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+      <nav class="headerP2">
```
Comment (Nyame-Wolf) [inline Index.html]: [OPTIONAL]I suggest you change this from a `nav` tag to `div` or any other tag. Nav stands for navigation. It is very confusing having these in the middle of the page for screen readers.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi there @Zuz4n ! , 


Great job introducing the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some issues that still need to be addressed for your project to shine :sparkles:.

## Highlights

- Great work using the correct GitHub Flow :100: 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  


 ## Required Changes ♻️

 There are some issues that you still need to work on, but you are nearly there! 
_For more information, check the comments under the review :arrow_down:_


## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Debas-31 — APPROVED** [review]
> ## Hi @Zuz4n   :wave:, 

# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**